### PR TITLE
PVS proxy enhancements

### PR DESCRIPTION
**Pull request by robhoes**

### CODE REVIEW

**Comment by robhoes** [comment]
@jonludlam I've updated this now.

File: ocaml/xapi/message_forwarding.ml [hunk 1]
```diff
@@ -3906,5 +3906,14 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
...
 
+	module PVS_cache_storage = struct
+		let create ~__context ~site ~sR ~size =
+			info "PVS_cache_storage.create";
+			Local.PVS_cache_storage.create ~__context ~site ~sR ~size
+
+		let destroy ~__context ~self =
+			info "PVS_cache_storage.destroy";
+			Local.PVS_cache_storage.destroy ~__context ~self
+	end
```
Comment (jonludlam) [inline ocaml/xapi/message_forwarding.ml:3917]: You could just `include` this module

File: ocaml/xe-cli/bash-completion [hunk 2]
```diff
@@ -105,14 +105,14 @@ _xe()
...
-                      || /^pvs-farm-/ { print $1 "-" $2 }
+                      || /^pvs-site-/ { print $1 "-" $2 }
                          $0 !~ /^host-cpu-/\
                       && $0 !~ /^host-crashdump-/\
                       && $0 !~ /^gpu-group-/\
                       && $0 !~ /^vgpu-type-/\
                       && $0 !~ /^pvs-server-/\
                       && $0 !~ /^pvs-proxy-/\
-                      && $0 !~ /^pvs-farm-/ { print $1 }')-list
+                      && $0 !~ /^pvs-site-/ { print $1 }')-list
```
Comment (jonludlam) [inline ocaml/xe-cli/bash-completion:115]: Ah, missed this! thanks!

File: ocaml/xapi/message_forwarding.ml (hunk 1 above)
Comment (robhoes) [inline ocaml/xapi/message_forwarding.ml:3917]: That's true now, but I think that we'll need to actually forwarding in here when the stubs are filled in.

Comment (jonludlam) [inline ocaml/xapi/message_forwarding.ml:3917]: Ah, that makes sense.

**Pull request merged**